riverdriver/riverpgxv5/internal/dbsqlc/river_leader.sql

```diff
@@ -9,16 +9,17 @@ CREATE UNLOGGED TABLE river_leader(
 
 -- name: LeaderAttemptElect :execrows
 INSERT INTO /* TEMPLATE: schema */river_leader (leader_id, elected_at, expires_at)
-    VALUES (@leader_id, coalesce(sqlc.narg('now')::timestamptz, now()), coalesce(sqlc.narg('now')::timestamptz, now()) + @ttl::interval)
+    -- @ttl is inserted as 
```

Comment:
not sure what this comment is meant to convey

Reply:
Ah doh, I started a comment, but didn't finish it. I just put this more expounded version in instead:

``` sql
-- @ttl is inserted as as seconds rather than a duration because `lib/pq` doesn't support the latter
```